### PR TITLE
Fix for cmdline in Routing module

### PR DESCRIPTION
**Pull request by mchf**
bnc#945947

### CODE REVIEW

File: src/modules/Routing.rb [hunk 1]
```diff
@@ -289,6 +290,24 @@ def Write
...
+      # work around bnc#19476
+      if device == ANY_DEVICE
+        filename = ROUTES_FILE
+
+        return SCR.Write(path(".target.string"), filename, "")
+      else
+        filename = "#{ROUTES_DIR}/ifroute-#{device}"
+
+        return SCR.Execute(path(".target.remove"), filename) if FileUtils.Exists(filename)
+        return true
```
Comment (jreidinger) [inline src/modules/Routing.rb]: more logic way for me is to quick exit for non exist file like:
```
return true unless FileUtils.Exists(filename)

SCR.Execute(path(".target.remove"), filename)
```

File: test/ifroute_test.rb [hunk 2]
```diff
@@ -81,6 +81,8 @@
 end
 
 describe "Routing#write_routes" do
+  Yast.import "FileUtils"
```
Comment (jreidinger) [inline test/ifroute_test.rb]: this should not be needed as it is already imported in Routing or I overlook something?

**Comment by mvidner** [comment]
LGTM, thanks!

**Comment by mchf** [comment]
Thanks for reviews

**Pull request merged**